### DISCUSSION TITLE
pixie pixel clustering that seperates background

### DISCUSSION THREAD

**Discussion by mezwick** [angelolab/ark-analysis#1149] [Q&A]
Hi,

PIXIE is doing a great job clustering my tissue regions, but i find it does not ascribe a single 'background' pixel metacluster (for pixels where there are little to no markers) and i end up with a rather colourful 'noise' background. For what is actually an ablated region containing no tissue. 

A bit like so...
![image](https://github.com/user-attachments/assets/e6af3730-0a1b-4042-9962-60ccb2853ebf)

I wondered if this happens because of the way pixie values the relative contribution of pixel intensities across channels. So when they are all low except for spruious signal (hot pixels, random noise etc) it leads to strange behaviour and pixels ascribed to clusters? 

Anyway... any tricks to encourage seperate clustering of background?

Thanks!
Merrick

**Comment by cliu72** [comment]
Hi @mezwick! You're right that this happens because of the way Pixie pre-processes the data to take into account the relative contribution of intensities (we found that this helps a lot for phenotyping). Pixie was built with the assumption that the images have been background subtracted and noise removed already. I would suggest creating a background mask, then masking all your images with this background mask (effectively setting all the pixels that are background to be 0). Then, feed those images into Pixie. One way to create this background mask is to take all pixels below a certain threshold. Or if you have a background channel, you could also use that to create a background mask. Hope this helps!
